index.ts

```diff
@@ -382,6 +387,12 @@ exports.main = async (context, req, res): Promise<void> => {
                 }
             }
         }
+        //Response is a buffer, expects a string.
+        //Sending the response should start the termination of the function, which is the main goal.
+        response.setStatusCode(200)
+        response.setHeader('content-type', 'application/json')
+        response.send('')
```

Comment:
request.end() doesn't work here? :)

Reply:
From what I tested it works to end the function but still prints a red "invocation error" on the function, which is not ideal.